src/components/spreadsheet/utils/config-tables.js

```diff
@@ -2348,15 +2372,9 @@ export const TABLES_DEFINITIONS = {
             {
                 id: 'RegulationTypeText',
                 field: 'RegulationTypeText',
-                ...defaultTextFilterConfig,
+                ...defaultEnumFilterConfig,
                 editable: isEditable,
                 cellStyle: editableCellStyle,
-                valueGetter: (params) =>
-                    params.data.RegulationTypeText ??
-                    (params.data?.regulatingTerminalVlId ||
-                    params.data?.regulatingTerminalConnectableId
-                        ? REGULATION_TYPES.DISTANT.id
-                        : REGULATION_TYPES.LOCAL.id),
```

Comment:
why do we need to remove this ?

Reply:
Because, now, for the calculated columns, this is handled during the formatting of the equipment data. As a result, we bypass the need for value getters and avoid issues related to validations in editing mode.